src/components/Inputs/RichTextEditor/custom-extensions/ExtendedImage.ts

```diff
@@ -57,25 +53,23 @@ export default Image.extend({
               if (!coordinates) return;
 
               for (const image of images) {
-                onImageUpload(image).then((item) => {
-                  if (!item) return;
-                  const node = schema.nodes.image.create({
-                    src: item.b64,
-                    alt: item.url,
+                onImageUpload(image)
+                  .then((item) => {
+                    if (!item) return;
+                    // TODO: is this still used?
```

Comment:
> is this still used?

Yeah we're still using the node that it creates here, but it looks like you remove the transaction that uses that newly created node, both here and in the function further down. These should be re added

Reply:
added the removed code back